usl_pipeline/usl_lib/usl_lib/storage/metastore.py

```diff
@@ -35,21 +39,33 @@ class StudyArea:
     elevation_min: Optional[float] = None
     elevation_max: Optional[float] = None
 
+    def as_header(self) -> geo_data.ElevationHeader:
+        """Represents the study area as a header describing a raster space."""
+        return geo_data.ElevationHeader(
+            col_count=self.col_count,
+            row_count=self.row_count,
+            x_ll_corner=self.x_ll_corner,
+            y_ll_corner=self.y_ll_corner,
+            cell_size=self.cell_size,
+            nodata_value=-9999.0,
+            crs=rasterio.CRS.from_string(self.crs) if self.crs is not None else None,
+        )
+
     def create(self, db: firestore.Client) -> None:
         """Creates a new study area in the given DB. Only writes non-None attributes.
 
         Args:
           db: The firestore database client to use for the read.
         """
-        db.collection(STUDY_AREAS).document(self.name).create(self._as_dict())
+        self.get_ref(db, self.name).create(self._as_dict())
 
     def set(self, db: firestore.Client) -> None:
         """Creates a new study area in the given DB. Only writes non-None attributes.
```

Comment:
I guess it's "Creates or updates" in the doc for "set" method?

Reply:
`create()` only creates and raises an error if the document already exists
`set()` creates or replaces if it already exists

Reply:
fixed docstring